### PR TITLE
LaurAlejandraC/Week1/UnitTest

### PR DESCRIPTION
**Pull request by LaurAlejandraC**
Add solution to Unit Test katas.

### CODE REVIEW

File: Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario1Test.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    @Test
+    public void transfer(){
+        AccountServiceImpl accountService = new AccountServiceImpl(accountRepository);
+
+        try{
+            TransferResponse transferResponse = accountService.transferMoney(currentAccount, savingsAccount, 2500);
+            Assert.assertTrue(transferResponse.getCurrentAccount().getBalance() == 500 && transferResponse.getSavingAccount().getBalance() == 4000);
+        }
+        catch (Exception ex){
```
Comment (estigma88) [inline Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario1Test.java]: You shouldn't have a try - catch structure in a unit test (usually). Your unit test should be focused on one flow, it means, if you have one flow where an exception is expected, you should create a different unit test with that flow. You can use @Test(throw=exception) to validate that an exception was thrown, or for more advanced cases, @Rule and ExpectedException class. It is fine to declare the exception in the sign method using throws word if for that flow doesn't matter the exception.

File: Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario2Test.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        currentAccount = new Account("User", 100, AccountType.CURRENT);
+        savingsAccount = new Account("User", 200, AccountType.SAVING);
+        accountRepository = new AccountRepositoryImpl(new ArrayList<>(Arrays.asList(currentAccount, savingsAccount)));
+    }
+
+    @Test
+    public void transferException(){
+        AccountService accountService = new AccountServiceImpl(accountRepository);
+
+        assertThatExceptionOfType(AccountException.class)
```
Comment (estigma88) [inline Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario2Test.java:34]: Awesome

**Review by estigma88 — CHANGES_REQUESTED** [review]

File: Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario1Test.java (hunk 1 above)
Comment (chejunior) [reply Week1/LaurAlejandraC/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/Scenario1Test.java]: I agree, I know that you have made another test case to check the exception and that was I expected, so please remove the "try/catch" clause and that's it :)

**Review by chejunior — APPROVED** [review]

**Review by estigma88 — APPROVED** [review]
Good job. Some comments to take into account next time

**Pull request merged**